### PR TITLE
Avoid negative space in \bmod.

### LINKED ISSUE #982: \bmod in inline math mode adds extra following space
Test file: https://cdn.rawgit.com/akalin/98bc47f5e54f7529462f12a1d7336858/raw/28a50452296c9d7bd4dc9e5023056cf64412a913/test-0.8.3.html

Using \bmod in inline math mode seems to add some extra following space, which seems like a bug. Not sure if it's a problem with other binary operators.

### PR DESCRIPTION
**Pull request by edemaine**
This is a workaround for #836 for the command case of using \bmod, without changing any spacing (and making the code and generated output slightly more efficient).  In particular, fixes #982.

### CODE REVIEW

**Comment by kevinbarabash** [comment]
Looks like the `mod` screenshots need to be regenerated.

File: src/functions/mod.js [hunk 1]
```diff
@@ -12,13 +12,16 @@ const htmlModBuilder = (group, options) => {
     const inner = [];
 
     if (group.value.modType === "bmod") {
-        // “\nonscript\mskip-\medmuskip\mkern5mu”
+        // “\nonscript\mskip-\medmuskip\mkern5mu”, where \medmuskip is
+        // 4mu plus 2mu minus 1mu, translates to 1mu space in
+        // display/textstyle and 5mu space in script/scriptscriptstyle.
```
Comment (kevinbarabash) [inline src/functions/mod.js]: Took me a bit to grok what's going on, but it makes sense.

**Comment by edemaine** [comment]
Hmm, seems there's more investigation to do.  Here's the diff between old and new screenshots (green is new):

mod-firefox-diff

More clear is to look at the texdiff.  Before this PR is near-perfect in the top left:

mod-old

After this PR is worse:

mod-new

**Comment by edemaine** [comment]
Turns out I just failed to properly add the muspace CSS rule.  (Alternatively, we could add 1mu as a manual style value instead of using a CSS rule.)  I believe the Mod screenshots are now unaffected; this will be confirmed once Travis finishes.

**Pull request merged**